services/grpc/grpc.js

```diff
@@ -19,15 +19,15 @@ class ZapGrpc extends EventEmitter {
   static INITIAL_STATE = {
     options: {},
     services: {},
-    subscriptions: [],
+    subscriptions: {},
   }
 
-  static SUBSCRIPTIONS = [
-    'subscribeInvoices',
-    'subscribeChannelGraph',
-    'subscribeTransactions',
-    'subscribeGetInfo',
-  ]
+  static SUBSCRIPTIONS = {
+    invoices: 'subscribeInvoices',
+    transactions: 'subscribeChannelGraph',
+    channelgraph: 'subscribeTransactions',
+    info: 'subscribeGetInfo',
+  }
```

Comment:
Just a thought. Do we really need this to be hardcoded? I'm thinking we can use a more dynamic approach. like `registerService` and pre-register those 4. This eventually should work better with LND API versioning. Like for instances soon we will be adding `scb` to a list of subscriptions.

Reply:
I've added a basic `registerSubscription` method that can be used to setup the mapping in a slightly more dynamic way.